js-src/src/app/games.component.html

```diff
@@ -1,4 +1,15 @@
 <div>
+    <h5>Game mode: </h5>
+
+	<div class="row">
+		<div class="co-sm-2">
+			<button type="button" class="btn btn-primary" (click)="easy()">Easy</button>
+			<span></span>
+			<button type="button" class="btn btn-primary" (click)="difficult()">Difficult</button>
+		</div>
+	</div>
+</div>
+<div>
     <h4>Games: </h4>
 
```

Comment:
the list games / search for games list should also display the difficulty if possible (not super important)